js/slider.js

```diff
@@ -0,0 +1,21 @@
+
+var slideIndex = 0;
+showSlides();
+
+function showSlides() {
+  var i;
+  var slides = document.getElementsByClassName("mySlides");
```

Comment:
We can make this `const` and also move it outside the function, so that we don't query the DOM everytime this function runs.